components/user/Tabs/WeeklyContent.tsx

```diff
@@ -29,42 +17,38 @@ export default function WeeklyContent({
       <WeeklyMetricCard
         title="Blocks Mined"
         metric={weeklyMetrics.metrics.blocks_mined}
-        metricValueMax={getWeeklyMetricMax(
-          API.EventType.BLOCK_MINED,
-          metricsConfig
-        )}
+        metricValueMax={metricsConfig.weekly_limits[API.EventType.BLOCK_MINED]}
```

Comment:
Nit, but I'd find it more readable if you did something like:

```
const {
  [EventType.BLOCK_MINED]: _blockMinedMetric
  [EventType.BUG_CAUGHT]: _bugsCaughtMetric
} = metricsConfig.weekly_limits
```

But that's obviously subjective -- I just find the current structure to be not very DRY.

Reply:
Updated this, let me know if this looks like how you mean!